src/test/java/io/confluent/examples/streams/WordCountLambdaIntegrationTest.java

```diff
@@ -15,79 +15,69 @@
  */
 package io.confluent.examples.streams;
 
-import io.confluent.examples.streams.kafka.EmbeddedSingleNodeKafkaCluster;
-import org.apache.kafka.clients.consumer.ConsumerConfig;
-import org.apache.kafka.clients.producer.ProducerConfig;
 import org.apache.kafka.common.serialization.LongDeserializer;
 import org.apache.kafka.common.serialization.Serde;
 import org.apache.kafka.common.serialization.Serdes;
 import org.apache.kafka.common.serialization.StringDeserializer;
 import org.apache.kafka.common.serialization.StringSerializer;
-import org.apache.kafka.streams.KafkaStreams;
 import org.apache.kafka.streams.KeyValue;
 import org.apache.kafka.streams.StreamsBuilder;
 import org.apache.kafka.streams.StreamsConfig;
+import org.apache.kafka.streams.TopologyTestDriver;
 import org.apache.kafka.streams.kstream.KStream;
 import org.apache.kafka.streams.kstream.KTable;
 import org.apache.kafka.streams.kstream.Produced;
 import org.apache.kafka.test.TestUtils;
-import org.junit.BeforeClass;
-import org.junit.ClassRule;
 import org.junit.Test;
 
 import java.util.Arrays;
 import java.util.List;
+import java.util.Map;
 import java.util.Properties;
 import java.util.regex.Pattern;
+import java.util.stream.Collectors;
 
+import static io.confluent.examples.streams.IntegrationTestUtils.mkEntry;
+import static io.confluent.examples.streams.IntegrationTestUtils.mkMap;
```

Comment:
Could we use `mkMap` in `code/kafka/clients/src/main/java/org/apache/kafka/common/utils/Utils.java` instead? Potential we could avoid double implementing the same feature.

Reply:
Huh, I thought that was an internal class. It certainly should be. This means we should have had a KIP to propose the mkMap method for Utils.

More generally, it's not good "hygine" to depend on another project's utils, even if they accidentally fail to keep them private. Kafka Client's "job" is to provide the client libraries, not to provide a way to make maps with varargs, so we shouldn't expect this to be a supported API.

Especially for a case like this, where the logic is so trivial, we should strongly prefer to copy the code, rather than introduce another external dependency. Every line of code is a maintenance burden, but every external dependency is like 10 maintenance burdens.